packages/ui-components/src/types.tsx

```diff
@@ -75,4 +75,6 @@ export interface Message {
     message: string;
     /** The timestamp of the message */
     timestamp: string;
+    /** Whether the message has been edited */
+    edited?: boolean;
```

Comment:
Rather than explicitly storing an edited flag, it would be more conventional to have a pair of `created_at` and `updated_at` properties and simply treat the message as edited when the two don't match. Then whenever an update is made you update `updated_at`

Reply:
I have added an updated_at which when set we show something and used it so that when one hovers the edited status they can see when it was last edited in the tooltip.

Reply:
In this case should it be with underscores as it is going to be used in the file saved or should it be camel case?

Reply:
probably underscores since it's the data model that will be saved in the JSON directly